feature-libs/commerce-quotes/components/commerce-quotes-action-links/commerce-quotes-action-links.component.html

```diff
@@ -0,0 +1,13 @@
+<section>
+  <!--
+    TODO: Does should be always new cart, or link to existing cart?
+    If first option, we need to fire clear cart action before 
+    navigation action will be ended.
+    -->
```

Comment:
As I remember this link should clear cart before navigation end.

Reply:
Yup, we should clear all data related to previous cart before we reach cart view (with in result will be empty tho)